modules/ppcp-button/resources/js/modules/ContextBootstrap/CartBootstap.js

```diff
@@ -15,6 +15,8 @@ class CartBootstrap {
         this.render();
 
         jQuery(document.body).on('updated_cart_totals updated_checkout', () => {
+
+            console.log(document.querySelector(this.gateway.button.wrapper), document.querySelector(this.gateway.hosted_fields.wrapper));
```

Comment:
Can we remove this console log here?

Reply:
done, thanks d3b8e04135d610e701360b9f72fe3fb0cbe1e3ad